cli/src/semdep/find_lockfiles.py

```diff
@@ -50,12 +50,9 @@ def find_single_lockfile(
             manifest_pattern = LOCKFILE_TO_MANIFEST[lockfile_pattern]
             manifest_path = path / manifest_pattern if manifest_pattern else None
             if lockfile_path.exists():
-                return lockfile_path, parse_lockfile_str(
-                    lockfile_path.read_text(encoding="utf8"),
+                return lockfile_path, parse_lockfile_path(
```

Comment:
Should we move manifest_path inside parse_lockfile_path?

i.e. is there any reason anything other than parse_lockfile_path need to know LOCKFILE_TO_MANIFEST and how to go from the lockfile to the manifest file? Looks like we basically have to repeat all these none checks (including line 51 above) every time we call this function

Reply:
I'm not completely sure what change you're suggesting. There's _definitely_ room to clean up some of the `None` checks in the code, but my inclination is just to get this merged for now.